src/components/MainSideBar/AddAccountButton.js

```diff
@@ -11,7 +11,7 @@ import { rgba } from 'styles/helpers'
 
 const PlusWrapper = styled(Tabbable).attrs({
   p: 1,
-  cursor: 'pointer',
+  cursor: 'default',
```

Comment:
`default` is already the default so we can remove the whole line

Reply:
done